src/test/java/puppy/registration/RegistrationHappyFlowTest.java

```diff
@@ -26,7 +26,12 @@ public void registrationHappyFlowTest() {
                 .loginWithTestAccount();
         dashboardPage
                 .checkDashboardIsDisplayed();
-        loginPage
-                .deleteTestAccount();
+        basePage
```

Comment:
You can call this directly from dashboard page since the method is inherited from base page. No need to call BasePage directly